esb/plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/AbstractMediator.java

```diff
@@ -156,6 +166,47 @@ public AbstractMediator(View view) {
 		instance=this;
 	}
 
+	/**
+	 * 
```

Comment:
Put the proper full Javadoc comment, or else, do not put Javadoc comments for obvious/self-describing methods. Fix in all places.